sag_ji3.par

```diff
@@ -1,118 +1,118 @@
--38.061940350	\!~	45/46	�	-19.9
--37.813682272	.~!)	524288/535815	.h	-19.6
--37.325763791	'(!(	8019/8192	'd	-19
--36.588490003	~!)	48/49	h	-18.4
--35.118091464	.(!	49/50	.j	-18
--34.398252906	(!	45056/45927	j	-17
--32.446578982	!/	54/55	k	-16.6
--31.470742020	'(!	55/56	'j	-16.4
--30.982823539	)!)	56/57	�	-16
--30.006986577	.!)	3584/3645	.t	-15
--28.543231135	!)	63/64	t	-14
--26.103638730	'!)	2066715/2097152	't	-13.4
--24.885981586	)\!	40960/41553	�	-13
--23.955448449	.\!	524288/531441	.\	-12.4
--23.176127844	')\!	76/65	'�	-12
--22.688209363	\!	80/81	\	-11
--20.248616959	)!~	19456/19683	�	-10.7
--19.760698478	'\!	2025/2048	'\	-10
--18.784861516	~~!	98/99	�	-9
--17.321106073	!~	729/736	z	-8
--15.369432149	~!(	4096/4131	a	-7
--13.417758226	'~!(	135/136	'a	-6.6
--12.441921264	)~!	143/144	�	-6.4
--11.954002783	.)!(	2816/2835	.i	-6
--10.978165821	)!(	891/896	i	-5
--9.0264918973	~!	2176/2187	s	-4.6
--8.0506549354	.!(	224/225	.c	-4.2
--7.0748179735	'~!	255/256	's	-4
--6.5868994926	!(	5103/5120	c	-3
--4.6352255689	)!	512/513	;	-2
--2.6835516451	.!	32768/32805	.	-1.3
--1.7077146833	')!	1215/1216	';	-1
--1.2197962023
-1.2197962023	.)|	1216/1215	."	1
-1.7077146833	'|	32805/32768	'	1.3
-2.6835516451	)|	513/512	"	2
-4.6352255689	|(	5120/5103	r	3
-6.5868994926	.~|	256/255	.$	4
-7.0748179735	'|(	225/224	'r	4.2
-8.0506549354	~|	2187/2176	$	4.6
-9.0264918973	)|(	896/891	*	5
-10.978165821	')|(	2835/2816	'*	6
-11.954002783	)~|	144/143	�	6.4
-12.441921264	.~|(	136/135	.g	6.6
-13.417758226	~|(	4131/4096	g	7
-15.369432149	|~	736/729	~	8
-17.321106073	~~|	99/98	�	9
-18.784861516	./|	2048/2025	./	10
-19.760698478	)|~	19683/19456	�	10.7
-20.248616959	/|	81/80	/	11
-22.688209363	.)/|	76/75	.�	12
-23.176127844	'/|	531441/524288	'/	12.4
-23.955448449	)/|	41553/40960	�	13
-24.885981586	.|)	2097152/2066715	.f	13.4
-26.103638730	|)	64/63	f	14
-28.543231135	'|)	3645/3584	'f	15
-30.006986577	)|)	57/56	�	16
-30.982823539	.(|	56/55	.?	16.4
-31.470742020	|\	55/54	y	16.6
-32.446578982	(|	45927/45056	?	17
-34.398252906	'(|	50/49	'?	18
-35.118091464	~|)	49/48	p	18.4
-36.588490003	.(|(	8192/8019	.q	19
-37.325763791	'~|)	535815/524288	'p	19.6
-37.813682272	/|~	46/45	�	19.9
-38.061940350	(|(	45/44	q	20
-39.765356196	~|\	16767/16384	�	20.8
-40.253274677	.//|	128/125	.=	21
-41.717030120	//|	6561/6400	=	22
-44.156622525	'//|	39/38	'=	23
-45.620377967	)//|	6656/6480	�	24
-47.572051891	/|)	36/35	n	25
-49.523725815	(|~	176/171	�	25.8
-50.011644296	'/|)	59049/57344	'n	26
-51.219540249	./|\	11264/10935	.^	26.6
-51.963318220	/|\	33/32	^	27
-53.914992143	(/|	4096/3969	�	28
-55.095545777	'/|\	1082565/1048576	'^	28.6
-55.866666067	)/|\	405/392	%	29
-57.818339991	.(|)	512/495	.@	30
-58.589460281	|\)	8680203/8388608	�	30.4
-59.770013914	(|)	729/704	@	31
-61.721687838	'(|)	23914845/23068672	'@	32
-62.465465809	.(|\	28/27	.m	32.4
-63.673361762	|\\	373977/360448	�	33
-64.161280243	(|\	8505/8192	m	33.2
-66.112954167	)|\\	885735/851968	�	34
-68.06462809
+-68.084530821955000	)!//	851968/885735		-34

```

Comment:
The lower bounds values increased accuracy to 15 digits, which was what George's JI calculator supported (not all of them seem to go to that level of accuracy, but at least one does). Hopefully that won't break Scala. Every one of them changed slightly too.